dev-app/android/app/build.gradle

```
@@ -258,6 +258,7 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:+"  // From node_modules
+    implementation 'com.facebook.soloader:soloader:0.10.4+'
```

What is this new dependency for?

---

It's added for solving this [issue](https://github.com/wix/Detox/issues/3487) which also happened in our case of running `"e2e:test:build:release"` test, where the app would crash on the beginning with error `"java.lang.UnsatisfiedLinkError: couldn't find DSO to load: libjscexecutor.so"`.

---

Can we change this to use a strict version instead of a dynamic one? I think using `0.10.4+` means it will just pick up the latest `0.10.*` version, [which currently is `0.10.5`](https://mvnrepository.com/artifact/com.facebook.soloader/soloader):

```suggestion
    implementation 'com.facebook.soloader:soloader:0.10.5'
```